com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -1084,7 +1084,7 @@ internal List<NetworkBehaviour> ChildNetworkBehaviours
                 }
 
                 m_ChildNetworkBehaviours = new List<NetworkBehaviour>();
-                var networkBehaviours = GetComponentsInChildren<NetworkBehaviour>(true);
+                var networkBehaviours = gameObject.GetComponentsInChildren<NetworkBehaviour>(true);
```

Comment:
wait... this does change the component search behavior? really? oof... is this well documented somewhere?
(and also, yeah, tests please if possible)

Reply:
Reading from the documentation, I don't get the impression that is the case, are we sure this is actually the case?

https://docs.unity3d.com/ScriptReference/Component.GetComponentsInChildren.html

Reply:
I double-checked against the engine source code and `GetComponentsInChildren<NetworkBehaviour>(true)` actually unfolds to `gameObject.GetComponentsInChildren<NetworkBehaviour>(true)` internally anyway.

Can be seen here out in public reference repo:
https://github.com/Unity-Technologies/UnityCsReference/blob/master/Runtime/Export/Scripting/Component.bindings.cs#L98

So, this is not a fix, perhaps a styling refactor.